docs/concepts/en-US/about_PSDocs_Azure_Badges.md

```diff
@@ -0,0 +1,74 @@
+# PSDocs_Azure_Badges
+
+## about_PSDocs_Azure_Badges
+
+## SHORT DESCRIPTION
+
+Describes how to insert template badges in to documentation.
+
+## LONG DESCRIPTION
+
+PSDocs allows external files to be included in Azure template documentation.
+Commonly this concept is used to include images that represent the validation status of the template.
+These images, are commonly referred to as _badges_.
+PSDocs can include badges that have been generated by an external validation tool.
+
+To include badge markdown:
+
+- Create a sub-directory called `.ps-docs` in the working path of PSDocs.
+This would normally be root directory (`$PWD`) of the repository where your Azure template are stored.
+- Create a file named `azure-template-badges.md` within the `.ps-docs` sub-directory.
+- When creating all files and folder use lower case names.
+
+The contents of this file is automatically inserted in generated output after the title but before description.
+
+### Include badges
+
+To include badge images use standard markdown syntax within the `azure-template-badges.md` file.
+For example:
+
+```markdown
+![label](https://image_uri)
+```
+
+To include badges images with a clickable link use standard markdown syntax:
+
+```markdown
+[![label](https://image_uri)](https://link_uri)
```

Comment:
Can we use the standard status Az DevOps badge such as [![Build Status](https://dev.azure.com/....)?

Reply:
Nice suggestion.